### DISCUSSION TITLE
Allow extensions to disable ratings

### DISCUSSION THREAD

**Discussion by johnsoncodehk** [microsoft/vsmarketplace#949] [General]
Hello VSCode/Marketplace team, I am the author of https://marketplace.visualstudio.com/items?itemName=Vue.volar. First of all, I would like to thank FOSS for sponsoring me. I am not sure whether your team will participate in the nomination. But I'm grateful anyway.

In the past I mentioned at https://github.com/vuejs/language-tools/issues/4227#issuecomment-2058199605 that I would like to be able to disable the extended rating functionality. For context, we started receiving many comments after the March 2024 release of Vue.volar 2.0.0.

Most of the comments are negative, while some are ignorant, inaccurate, or even offensive. Vue.volar has always been one of the most actively updated extensions on the Marketplace, we have been actively addressing all bug reports, negative comments do nothing to improve the extension, it does not provide the context or reproduction needed to solve the problem. On the contrary, every time I receive these comments it reduces my motivation to maintain this project. When this situation lasted for several months, it was close to a form of cyberbullying, and I suffered from depression and refused to browse the project's issues page.

Comments are a way to unilaterally punish extension developers. This is unfair to extension authors. Extension authors have the right to reject the noise. We should add an option to disable ratings for extensions.

Thank you for your consideration. 🙏

**Comment by benny-yamagata** [comment]
Disable ratings or comments? 

I'm fine with disabling comments, but ratings I think is a problem.
